data/obs-packaging/cc-oci-runtime/README.md

```diff
@@ -18,3 +18,24 @@ This script will update the sources to create ``cc-oci-runtime`` packages.
 
   * cc-oci-runtime.dsc
   * cc-oci-runtime.spec
+
+OBS
+---
+
+The script has two variables related to OBS so the CI can be used to push 
+changes to [Open Build Service](https://build.opensuse.org/)
+
+  * ``OBS_PUSH`` default false
+  * ``OBS_RUNTIME_REPO`` default "home:clearlinux:preview:clear-containers-staging/cc-oci-runtime"
+
+If you want to push your changes and trigger a new build of the runtime to
```

Comment:
Change to:
To push your changes and trigger a new build of the runtime to the OBS repo, set the variables in the environment running the script before calling ``update_runtime.sh`` as follows: